### PR TITLE
Fix style guide nits in tutorial

### PR DESCRIPTION
**Pull request by steve-hardman**
As request in the issue have incorporated the feedback from https://github.com/kubernetes/website/pull/41260#pullrequestreview-1632808581

This will fix https://github.com/kubernetes/website/issues/43179

### CODE REVIEW

File: content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html [hunk 1]
```diff
@@ -114,13 +114,13 @@ <h3>Create a new Service</h3>
...
-        {{< note >}}<p>If Docker is the container runtime, a minikube tunnel is needed. This is because Docker Desktop does not support ports. <br>
+        {{< note >}}<p>If you're running minikube with Docker Desktop as the container driver, a minikube tunnel is needed. This is because containers inside Docker Desktop are isolated from your host computer.<br>
 				<p>In a separate terminal window, execute:<br>
-				<code><b>$ minikube service kubernetes-bootcamp --url</b></code></p>
+				<code><b>minikube service kubernetes-bootcamp --url</b></code></p>
 				<p>The output looks like this:
 				<pre><b>http://127.0.0.1:51082<br>!  Because you are using a Docker driver on darwin, the terminal needs to be open to run it.</b></pre></p>
 				<p>Then use the given URL to access the app:<br>
-				<code><b>$ curl 127.0.0.1:51082</b></code></p>
+				<code><b>url 127.0.0.1:51082</b></code></p>
```
Comment (dipesh-rawat) [inline content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html:123]: Minor error!
```suggestion
				<code><b>curl 127.0.0.1:51082</b></code></p>
```

**Comment by steve-hardman** [comment]
Thanks @dipesh-rawat! I have applied your error fix. Please review now.

**Review by dipesh-rawat — COMMENTED** [review]
It appears that PR does address the outstanding feedback from [#41260](https://github.com/kubernetes/website/pull/41260/files), just as was requested. 

Thanks!
/lgtm

[Preview Page](https://deploy-preview-43187--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/expose/expose-intro/)

**Pull request merged**